packages/compiler/server/serverlib.ts

```diff
@@ -626,13 +663,16 @@ export function createServer(host: ServerHost): Server {
       } else {
         switch (node.kind) {
           case SyntaxKind.NamespaceStatement:
+            getTypeDetails(params);
```

Comment:
That should probably be in `addKeywordCompletion` and the other `add ...` instead and set the result to the `detail`